README.md

```diff
@@ -84,27 +84,36 @@ These control the size of the viewport in your app.
 
 ##### `imageHeight` & `imageWidth` (Number)
 
-The height and width of the image you wish to render.
+The height and width of the image you wish to render. They are not necessary
+if you use tiff.
 
 ##### `initialViewState` (object) [ONLY NECESSARY FOR `MicrsocopyViewer`]
 
 An object containing two things
+
 - `target` (Array) An `[x,y,0]` location in image coordinates of the image. The 0
   represents a hypothetical (and potentially future addition) of a third spatial dimension.
 - `zoom` (Number) The initial zoom level to render the image at
 
 ##### `minZoom` & `maxZoom` (Number)
 
 These control the max and min zoom sizes, generally the number of images `n` in your pyramid,
-ranging from `-n` (zoomed out) to `0`, the highest resolution.
+ranging from `-n` (zoomed out) to `0`, the highest resolution. They are not necessary
+if you use tiffs.
 
-##### `sliderValues` (Array) **POTENIAL FUTURE BREAKING CHANGES WITH NEW FEATURES**
+##### `sliderValues` (Array)
 
 An object containing slider (max/min) values for each channel:
-`{sliderValues:{name:value}, {name:value}, {name:value}}`
+`{sliderValues:{name:value, name:value, name:value}}`
 
-##### `colors` (Array)
+##### `colorValues` (Array)
 
-Again, this is an objecting matching the sliders and the data of the colors
+Again, this is an object matching the channel names and colors
 that you wish to map to a full range for displaying,
-`{colorValues:{name:[r,g,b]}, {name:[r,g,b]}, {name:[r,g,b]}}`
+`{colorValues:{name:[r,g,b], name:[r,g,b], name:[r,g,b]}}`
```

Comment:
This is more clear. Perhaps:
```suggestion
`{colorValues: {channel1: [r,g,b], channel2: [r,g,b], channel3: [r,g,b]}}`
```

Is this actually the value that is passed in, or should it just be?:
```
{channel1: [r,g,b], channel2: [r,g,b], channel3: [r,g,b]}
```